lib/api/modules/epoch/epoch.ts

```diff
@@ -13,8 +13,8 @@ see files:
 export class Epoch {
   client: PublicClient;
 
-  static _STARTING_TIMESTAMP = 1_663_224_162;
-  static _EPOCH_PERIOD_SECONDS = 1_296_000; //15 days in seconds
```

Comment:
this cant be merged, the ideia is dev start using the epoch from mainnet. this get close as possible to same code base

Reply:
you probably tested on dev env instead of prod one

Reply:
need to rollback this changes